### PR TITLE
externed the show function to be in nes56_cv_manager

### PR DESCRIPTION
**Pull request by harelon**
Signed-off-by: Harel <harel.games@gmail.com>
this is what i meant in the last patch

### CODE REVIEW

File: rcv/frame_handlers/frc_2019.py [hunk 1]
```diff
@@ -74,10 +73,9 @@ def prep_output_image(self):
             output = cv2.circle(output, self._points[1], 3, (0, 255, 0), -1)
         self._output_frame = output
 
-    def show_images(self):
+    def get_sliders(self):
         self.prep_output_image()
-        cv2.imshow("output", self._output_frame)
-        cv2.waitKey(1)
+        return [('original', self._frame), ('cleaned', self._cleaned_frame), ('output', self._output_frame)]
```
Comment (bazulay) [inline rcv/frame_handlers/frc_2019.py]: I don't think you ever need to show all 3 images, only the output

File: rcv/frame_handlers/frc_2019.py [hunk 2]
```diff
@@ -74,10 +73,9 @@ def prep_output_image(self):
             output = cv2.circle(output, self._points[1], 3, (0, 255, 0), -1)
         self._output_frame = output
 
-    def show_images(self):
+    def get_sliders(self):
```
Comment (bazulay) [inline rcv/frame_handlers/frc_2019.py]: This is a bad name for a function

Comment (harelon) [reply rcv/frame_handlers/frc_2019.py]: I agree but i didn't know how to call it

File: rcv/frame_handlers/frc_2019.py (hunk 1 above)
Comment (harelon) [reply rcv/frame_handlers/frc_2019.py]: When i wanted to know where the error in detection these were the images I needed

**Comment by harelon** [comment]
I thought that the frc_2019 shouldn't display itself, I thought it should be the nes56_cv_manager that shows the images that frc_2019 wants to show if he is told to in the configuration or not i thought the frame handler just analyze the frame and returns the data about it, the modified images and the points of the white line

File: rcv/frame_handlers/frc_2019.py (hunk 1 above)
Comment (bazulay) [reply rcv/frame_handlers/frc_2019.py]: You don't need all images, you can store either the original image, or the output, and log the data into the log file with the name of the stored image. so you don't need all images

**Comment by bazulay** [comment]
The frame handler handles each frame and calculates whatever is needed by the nes56_cv_manager.
So since showing a frame is an operation on the frame and it depends on the sequence of function calls on the frame handler, it looks to me much better to do it in the frame handler.
I suggest discarding this patch

**Comment by harelon** [comment]
But the nes56 closes all windows on exit so why wouldn't it also start showing the frames, i think one of the things needed by nes56_cv_manager is the images of its place progress

**Pull request closed without merging**